crates/bevy_winit/src/winit_config.rs

```diff
@@ -68,7 +68,11 @@ pub enum UpdateMode {
     /// Once the app has executed all bevy systems and reaches the end of the event loop, there is
     /// no way to force the app to wake and update again, unless a `winit` event (such as user
     /// input, or the window being resized) is received or the time limit is reached.
-    Reactive { max_wait: Duration },
+    Reactive {
+        /// The maximum time to wait before the event loop runs again. Note that if the duration is
```

Comment:
These should be line split, and the advice should be more precise. What do we need to set the value to in order to get an indefinite wait? 

In many ways, this is actually a feature, so we should demonstrate how to use it.

Reply:
Well the problem is that it's platform dependent. Some platforms store Instants as u32s, others store them as u64s so the answer is "dunno, you figure it out." :) That said, using Duration::MAX is effectively the same as waiting for ever so I made a note of that.